source/Mlos.Notebooks/SmartCacheCPP.ipynb

```diff
@@ -0,0 +1,498 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Connecting MLOS to a C++ application\n",
+    "\n",
+    "This notebook walks through connecting MLOS to a C++ application within a docker container.\n",
+    "We will start a docker container, and run an MLOS Agent within it. The MLOS Agent will start the actual application, and communicate with it via a shared memory channel.\n",
+    "In this example, the MLOS Agent controls the execution of the workloads on the application, and we will later connect to the agent to optimize the configuration of our application.\n",
+    "\n",
+    "The application is a \"SmartCache\" similar to the one in the SmartCacheOptimization notebook, though with some more parameters to tune.\n",
+    "The source for this example is in the `source/Examples/SmartCache` folder."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Building the application\n",
+    "\n",
+    "To build and run the necessary components for this example you need to create and run a docker image.\n",
+    "To that end, open a separate terminal and go to the MLOS main folder. Within that folder, run the following commands:\n",
+    "\n",
+    "1. [Build the Docker image](../../../documentation/01-Prerequisites.md#build-the-docker-image) using the [`Dockerfile`](../../../Dockerfile#mlos-github-tree-view) at the root of the repository.\n",
```

Comment:
```suggestion
    "1. [Build the Docker image](../../../documentation/01-Prerequisites.md#build-the-docker-image) using the [`Dockerfile`](../Dockerfile#mlos-github-tree-view) at the root of the repository.\n",
```

Reply:
Although, in glancing through things just now, it seems as though these relative links don't work when github renders the notebook for us:
https://github.com/microsoft/MLOS/blob/55999553d22cd16a782bd183db63c44be8501a39/source/Mlos.Notebooks/SmartCacheCPP.ipynb
So, we might just be better off using absolute links to github.io pages for the documentation.